projects.yaml

```diff
@@ -3,370 +3,406 @@ projects:
   pyacp:
     name: PyACP
     repository: https://github.com/ansys/pyacp
-    documentation: https://acp.docs.pyansys.com/version/dev
-    examples: https://acp.docs.pyansys.com/version/dev/examples/index.html
-    api: https://acp.docs.pyansys.com/version/dev/api/index.html
-    userguide: https://acp.docs.pyansys.com/version/dev/user_guide/index.html
     thumbnail: _static/thumbnails/pyacp.png
-    intro: Pythonic interface to Ansys Composite PrepPost (ACP)
+    documentation:
+      base: https://acp.docs.pyansys.com/version/dev
+      base_intro: Pythonic interface to Ansys Composite PrepPost (ACP)
```

Comment:
This should be a description. We could add this below the `repository` key since it is not a documentation link.